consul_exporter.go

```diff
@@ -207,15 +227,35 @@ func (e *Exporter) setMetrics(services <-chan []*consul_api.ServiceEntry) {
 	}
 }
 
+func (e *Exporter) setKeyValues() {
+	kv := e.client.KV()
+
+	pairs, _, err := kv.List(e.kvPrefix, &consul_api.QueryOptions{})
+	if err != nil {
+		log.Errorf("Error fetching key/values: %s", err)
```

Comment:
It's more common in Go to return here and get rid of the `else` clause completely then. Less indentation, fewer control structures...